molgenis-questionnaires/src/main/frontend/src/services/questionnaireService.js

```diff
@@ -137,11 +166,23 @@ export default {
 
   printContent: function (docTitle: string, content: Object) {
     let docDefinition = {
+      pageSize: 'LETTER',
```

Comment:
A4 is an international standard, Letter is mostly used in the US/Canada. Is A4 a better default for our mostly EU users?

Reply:
@ConnorStroomberg could your reply why letter is the preferred choice here?